db/seeds.rb

```
@@ -18,3 +18,14 @@
 end
 
 Rails.logger.debug '✅ Seed user created: admin@app.local'
+
+# Create a default event for development
+Event.find_or_create_by!(descriptor: 'Default Event') do |event|
+  event.run_number = 1
+  event.date = Date.today
```

RuboCop (1.75.5)

[convention] Do not use `Date.today` without zone. Use `Time.zone.today` instead.

(Rails/Date)